client/app/components/dialog-editor-field/dialog-editor-field.component.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+  'use strict';
+
+  angular.module('app.components')
+    .component('dialogEditorField', {
+      bindings: {
```

Comment:
do these bindings need to be two way?  

if so you would be better off using the life cycle hooks and `&` to broadcast events 
if not `<` denotes a one way binding